src/components/Pagination/__tests__/Wrapper.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Pagination } from '../'
+
+describe('<Pagination.Wrapper />', () => {
+  it('passes in an optional className', () => {
+    const $ = shallow(<Pagination.Wrapper className="test">_</Pagination.Wrapper>)
+    expect($.hasClass('test')).toBe(true)
+  })
+
+  it('renders with attributes', () => {
+    const $ = shallow(
+      <Pagination.Wrapper style={{ position: 'relative' }} ariaHidden="true">
+        _
+      </Pagination.Wrapper>
+    )
+    expect($.prop('style')).toEqual({
+      position: 'relative'
+    })
+    expect($.prop('ariaHidden')).toBe('true')
+  })
+
+  it('renders a defined node type', () => {
+    const $ = shallow(<Pagination.Wrapper node="article">_</Pagination.Wrapper>)
+    expect($.type()).toBe('article')
+  })
+
+  it('renders a nav by default', () => {
+    const $ = shallow(<Pagination.Wrapper>_</Pagination.Wrapper>)
+    expect($.type()).toBe('nav')
+  })
+
+  it('renders children', () => {
+    const $ = shallow(
+      <Pagination.Wrapper>
+        Test
+      </Pagination.Wrapper>
+    )
+    expect($.contains('Test')).toBe(true)
+  })
+})
```

Comment:
Need to add a test for the `aria-label` attribute - it should have the default described in my earlier comment and an override passed via a prop.

Reply:
done

Reply:
I have added a test on the wrapper for this however this is currently failing, please could you check over both the Link.jsx and Wrapper.jsx to see what is missing - on the App.jsx I have added in some test override values in just for now so please could you let me know cheers

```
  it.skip('renders the aria-label with a prop passed in as an override', () => {
    const $ = shallow(<Pagination.Wrapper ariaLabel="Test Override">_</Pagination.Wrapper>)
    expect($.prop('ariaLabel')).toBe('Test Override')
  })```

Reply:
Have fixed this now